2021-06-24 17:34:45,766 - create_public - ERROR - Error detected
Traceback (most recent call last):
  File "C:\Users\Webdom16\PycharmProjects\TestsCovid\COVID_19_ 9880\Cоздание_исследований\create_public.py", line 31, in genlog
    self.create_public()
  File "C:\Users\Webdom16\PycharmProjects\TestsCovid\COVID_19_ 9880\Cоздание_исследований\create_public.py", line 57, in create_public
    driver.find_element_by_css_selector("#itemForm\:tabView\:sex > tbody > tr > td:nth-child(1) > div > div.ui-radiobutton-box.ui-widget.ui-corner-all.ui-state-default > span").click()
AttributeError: 'NoneType' object has no attribute 'click'
